src/Messaging/test/MessagingBase.Test/Handler/DestinationPatternsMessageConditionTest.cs

```diff
@@ -132,7 +132,7 @@ private DestinationPatternsMessageCondition Condition(params string[] patterns)
 
         private IMessage MessageTo(string destination)
         {
-            return MessageBuilder.WithPayload(new byte[0]).SetHeader(
+            return MessageBuilder.WithPayload(Array.Empty<byte>()).SetHeader(
```

Comment:
Please add `using System;` to this file and then the PR should be ready to merge, thank you!